microsoft-edge/test-and-automation/devtools-protocol.md

```diff
@@ -0,0 +1,14 @@
+---
+description: Overview of Microsoft Edge DevTools Protocol.
+title: DevTools Protocol
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 11/03/2021
+ms.topic: reference
```

Comment:
Can we confirm that reference is API ref only? I think ms.topic should be conceptual.

Reply:
Changed to "conceptual".  For the value "reference", internal DCG reads "Any authored (read: not autogenerated) content placed under a Reference node in a ToC."  This article was in a small, 4-child Reference node until recently.  The page contains reference-type information about HTTP endpoints (URL path patterns).  The connotations of the word 'reference' are shifting or broadening from API Reference to include REST Endpoint Reference.  DCG only defines the values in terms of the TOC bucket label, "Reference", and mentions "autogenerated" vs. (manually) "authored" pages.